monitoring/values-tempo.yaml

```diff
@@ -13,8 +13,12 @@ tempo:
   reportingEnabled: false
   metricsGenerator:
     # -- If true, enables Tempo's metrics generator (https://grafana.com/docs/tempo/next/metrics-generator/)
-    enabled: false
-    remoteWriteUrl: "http://prometheus.monitoring:9090/api/v1/write"
+    enabled: true
+    remoteWriteUrl: "http://v4m-prometheus.monitoring:9090/api/v1/write"
```

Comment:
Hmmm, this contains a hard-coded namespace reference.  Is there any way to _easily_ make this "generic" and avoid that?  If not, let's leave it as-is for now but plan on fixing it in an upcoming sprint.  If someone bumps into it before then we can walk them through providing the correct value in their USER_DIR/monitoring/user-values-tempo.yaml file.